### PR TITLE
add logic for authorization with 2 tokens

### PR DESCRIPTION
**Pull request by nottoday2471**

### CODE REVIEW

File: src/helpers/getNewAuthTokens.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { AxiosError } from 'axios';
+
+import userService from '../api/user.service';
+import { AuthTokensType } from '../types/auth-tokens.type';
+
+export const getNewAuthTokens = async (
+  refreshToken: string | null,
```
Comment (RamboUA) [inline src/helpers/getNewAuthTokens.ts:7]:
```suggestion
  refreshToken?: string,
```

File: src/helpers/getNewAuthTokens.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  deniedAccessWay?: Function,
+  setError?: Function
+) => {
+  const res: AuthTokensType = await userService
+    .refreshTokens(refreshToken)
+    .catch(err => {
+      console.log(err);
```
Comment (RamboUA) [inline src/helpers/getNewAuthTokens.ts:14]: Is it something intended?
Better not to left console.logs if there no special need for it

File: src/helpers/retrieveUser.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        deniedAccessWay,
+        setError
+      );
+      const rawUser = await userService.retrieve(tokens.accessToken);
+      setState(rawUser);
+    } else {
+      setError(err);
+    }
+  }
+  // const rawUser = await userService
```
Comment (RamboUA) [inline src/helpers/retrieveUser.ts:29]: Do you still need this commented part? If so can you please add TODO: of what should be done with this commented code further?

**Review by RamboUA — CHANGES_REQUESTED** [review]
Great job, please check the comments below

**Review by BillyAleks — APPROVED** [review]

**Pull request merged**